pkg/authorization/registry/clusterpolicy/registry.go

```diff
@@ -121,23 +121,23 @@ func (s *simulatedStorage) DeletePolicy(ctx apirequest.Context, name string) err
 }
 
 type ReadOnlyClusterPolicy struct {
-	Registry
+	Registry Registry
```

Comment:
Ordinarily I say, "no, you can't go direct to etcd for this", but its only for one release and I'm gathering this is code we've had and basically already tested.